### PR TITLE
iCarousel: Add Xcode automatic fixes.

### PR DESCRIPTION
**Pull request by Shachlan**

### CODE REVIEW

File: iCarousel.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -424,6 +436,7 @@
 				821507491E57475B00C75711 /* Release */,
 			);
 			defaultConfigurationIsVisible = 0;
+			defaultConfigurationName = Release;
```
Comment (barakwei) [inline iCarousel.xcodeproj/project.pbxproj:439]: I'm not sure if this hurts you or not.

**Comment by barakwei** [comment]
Looks like the changes are just for Debug

**Comment by barakwei** [comment]
@Shachlan ping

**Comment by Shachlan** [comment]
@barakwei see line 282 - it's also in release.

Regarding `defaultConfigurationName`, I'm not sure that removing this will be beneficial, since these are auto-fixes requested by XCode.

File: iCarousel.xcodeproj/project.pbxproj (hunk 1 above)
Comment (barakwei) [reply iCarousel.xcodeproj/project.pbxproj:439]: I checked in other projects, this seems to be fine.

**Comment by barakwei** [comment]
You're right, everything here is fine.

**Pull request merged**